### DISCUSSION TITLE
Saving Files/Comments

### DISCUSSION THREAD

**Discussion by dulibrary** [community/community#27322] [Other Feature Feedback, Questions, & Ideas]
Hello! After Visual Studio finished updating yesterday, I am having trouble adding/saving a comment for the file. We also had to reset our password because the old one was no longer working. There used to be a + at the top of the file where I can add a comment about the changes I made, but I no longer see it. Anyone else experiencing this? Any and all help would be appreciated!

**Comment by jfullstackdev** [comment]
what I see here, I think so, is unstable release

https://stackoverflow.com/questions/53380875/visual-studio-sometimes-not-saving-files

but of course, not very sure of that
